bridges/snowbridge/pallets/inbound-queue/src/lib.rs

```diff
@@ -59,7 +59,7 @@ use xcm_executor::traits::TransactAsset;
 use snowbridge_core::{
 	inbound::{Message, VerificationError, Verifier},
 	sibling_sovereign_account, BasicOperatingMode, Channel, ChannelId, ParaId, PricingParameters,
-	StaticLookup,
+	StaticLookup as SnowBridgeStaticLookup,
```

Comment:
This may have been due to my previous work.. Can you check if it can be reverted now?

Reply:
reverted in https://github.com/paritytech/polkadot-sdk/pull/5995/commits/88265e59c283c83335e81f8115378efe174fb9a1